jax/experimental/jax2tf/README.md

```diff
@@ -386,16 +394,86 @@ jax2tf.convert(lambda x: jnp.sum(x, axis=0) / x.shape[0],
                polymorphic_shapes=["(v, _)"])(np.ones((4, 4)))
 ```
 
-## Caveats
+## Known issues
 
 ### Incomplete TensorFlow data type coverage
 
 There are a number of cases when the TensorFlow ops that are used by the
-jax2tf converter are not supported by TensorFlow for fewer data types than JAX.
+jax2tf converter are not supported by TensorFlow for the same data types as in JAX.
 There is an
 [up-to-date list of unimplemented cases](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/g3doc/primitives_with_limited_support.md).
 
-### Missing features
+There are two kinds of errors you may see. For the primitives in the
+[unimplemented cases](https://github.com/google/jax/blob/master/jax/experimental/jax2tf/g3doc/primitives_with_limited_support.md)
+that are shown to be undefined on all devices and for all execution modes
+(`eager`, `graph`, `compiled`), e.g., `lax.min` for booleans,
+the conversion typically uses a TensorFlow operator that is not
+registered for a certain data type:
+
+```python
+jax2tf.convert(lambda x: lax.min(x, x))(np.array([True]))
+
+>>> InvalidArgumentError: Value for attr 'T' of bool is not in the list of allowed values:
+>>>    bfloat16, half, float, double, uint8, int16, int32, int64;
+>>>    NodeDef: {{node Minimum}};
+>>>    Op<name=Minimum; signature=x:T, y:T -> z:T; attr=T:type,allowed=[DT_BFLOAT16, DT_HALF, DT_FLOAT, DT_DOUBLE, DT_UINT8, DT_INT16, DT_INT32, DT_INT64]> [Op:Minimum]
+```
+
+In the above cases, you should file a bug with JAX or TensorFlow, or consider
```

Comment:
Small suggestion: consider being a bit more specific in the recommendation. I think for cases where the TF dtype is not registered, the bug should likely go to TF. Or are you suggesting that JAX should implement a workaround where JAX will lower to a different dtype in the converter?

Reply:
I think that the solution depends on the case. In some cases the bug should clearly go to TF, e.g., `lax.min` working on `int32` but not `int8`. However, in practice it is still us who fix those bugs, and we are actively working on it. Before long, the remaining cases are those when it is not so clear that the TF op should be registered at a given type. Examples are: comparisons of complex numbers, min/max for booleans. In those cases XLA does work, and has picked a semantics, but TF may not want to go that far. In those cases we might have to add workarounds in the converter.